easybuild/easyconfigs/n/Nsight-Systems/Nsight-Systems-2021.5.1.eb

```
@@ -0,0 +1,24 @@
+easyblock = 'Binary'
+
+name = 'Nsight-Systems'
+version = '2021.5.1'
+
+homepage = 'https://developer.nvidia.com/nsight-compute'
+description = """NVIDIA® Nsight™ Systems is a system-wide performance analysis tool designed to visualize an
+application’s algorithm, help you select the largest opportunities to optimize, and tune to scale efficiently across
+any quantity of CPUs and GPUs in your computer; from laptops to DGX servers."""
+
+toolchain = SYSTEM
+
+# Download requires login https://developer.nvidia.com/downloads
```

```suggestion
download_instructions = """Download %(name)s %(version)s from the NVIDIA website, which requires a NVIDIA developer account:
1. Go to https://developer.nvidia.com/gameworksdownload
2. Find %(name)s %(version)s
3. Download the "Nsight Systems 2021.5 (Linux Host .run Installer)". This may require logging in to your NVIDIA developer account.
"""
```

I think this syntax is correct...

---

okay this is now fixed